code/modules/mob/living/carbon/alien/humanoid/update_icons.dm

```diff
@@ -112,26 +112,36 @@
 	if(r_store)		r_store.screen_loc = ui_storage2
 	if(update_icons)	update_icons()
 
+/mob/living/carbon/alien/humanoid/update_inv_hand(index, var/update_icons = 1)
```

Comment:
Monkeys and aliens need special code treatment because their overlay system is different from humans (they use images as overlays, while humans use /obj/Overlays objects that can handle infinite hands just fine). See martians